.travis.yml

```diff
@@ -1,10 +1,22 @@
 sudo: true
 
+addons:
+  apt:
```

Comment:
Can this be switched to `yum`. We are primarily focused on centos/7 and RHEL7 (when we can) to keep the content focused on learning Ansible rather filling the docs and content with all the differences and caveats related to the various flavors of Linux.

Reply:
I think Travis only gives you Ubuntu or osx https://docs.travis-ci.com/user/reference/overview/

Reply:
I don't think we need to be OS focused in the travis yml file like we are in the lessons.  This is fine.

Reply:
@tima 
Yea, it's just for sanity check that Vagrantfile is basically working. Vagrant will spawn a CentOS inside containers anyway. So it doesn't really matter what the host OS is. I couldn't make it run VM, b/c they don't allow/support nested virtualization. Pure LXC didn't work either (I decided to skip troubleshooting this). The difference with VM flow is minimal, would allow testing most of things.
After all, it's not about testing host, right?

Reply:
@webknjaz Agreed, Tima and i chatted about this last night.  It'll get merged.

Thanks for your work on this. :)